blackbox/handlers/notifiers/telegram.py

```diff
@@ -0,0 +1,49 @@
+import telebot
+from telebot.apihelper import ApiTelegramException
+
+from blackbox.handlers.notifiers._base import BlackboxNotifier
+from blackbox.utils.logger import log
+
+
+STRING_LIMIT = 2000  # Limit output
+CHECKMARK_EMOJI = "\U00002705"  # ✔
+FAIL_EMOJI = "\U0000274C"  # ❌
+WARNING_EMOJI = "\U000026A0"  # ⚠
```

Comment:
The PEP8 states that an inline comment "should be separated **by at least two spaces** from the statement".

That means, there's no rule against lining these up, and I think it looks quite nice and tidy.

```suggestion
STRING_LIMIT = 2000  # Limit output
CHECKMARK_EMOJI = "\U00002705"  # ✔
FAIL_EMOJI = "\U0000274C"       # ❌
WARNING_EMOJI = "\U000026A0"    # ⚠
```